packages/sdk/src/evm/common/currency.ts

```diff
@@ -192,3 +192,22 @@ export async function normalizeAmount(
   const decimals = await contractWrapper.readContract.decimals();
   return utils.parseUnits(AmountSchema.parse(amount), decimals);
 }
+
+export function toEther(amount: Amount): string {
+  return utils.formatEther(AmountSchema.parse(amount));
+}
+
+export function toWei(amount: BigNumberish): BigNumber {
+  return utils.parseEther(AmountSchema.parse(amount));
+}
+
+export function toUnits(amount: Amount, decimals: number): BigNumber {
+  return utils.parseUnits(AmountSchema.parse(amount), decimals);
+}
+
+export function toDisplayValue(
+  amount: BigNumberish,
+  decimals: number,
+): string {
+  return utils.formatUnits(AmountSchema.parse(amount), decimals);
```

Comment:
Same here

Reply:
Here `decimals` should be `BigNumberish` like the interface `function parseUnits(value: string, unitName?: BigNumberish):` I will fix it